torchci/components/additionalTestInfo/TestInfo.tsx

```diff
@@ -90,6 +103,15 @@ function TDInfo({
   }
 
   if (Object.keys(info).length == 0) {
+    if (isPending(jobs)) {
+      return (
+        <div>
+          Workflow is still pending. Consider generating info in the
+          corresponding tab. If you have already done this, no test files were
+          excluded or there was trouble parsing data
+        </div>
```

Comment:
nit: There's a ton of places this message had to be copy/pasted. Can it be extracted out?

Reply:
Tried with middling success